qbft/spectest/tests/controller/decided/invalid_full_data.go

```diff
@@ -1,8 +1,35 @@
 package decided
 
-import "github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+	"github.com/herumi/bls-eth-go-binary/bls"
+)
 
 // InvalidFullData tests signed decided with an invalid full data field
 func InvalidFullData() tests.SpecTest {
-	panic("implement")
+	ks := testingutils.Testing4SharesSet()
+
+	msg := testingutils.TestingCommitMultiSignerMessageWithHeight(
```

Comment:
@GalRogozinski this kind of makes it a weird situation no? A decided message was received but the node thinks its invalid so it is basically stuck?

Reply:
Is this the expected behavior we expect?

Reply:
I see
If I understand correctly QBFT doesn't know what "invalid" data is. It just wants to reach a consensus on some data.
Also SSV protocol only has "slashable" checks. It doesn't know if the data is valid for the underlying blockchain protocol (ethereum in our case).

So maybe we should change "invalid" to "arbitrary"? Because "invalid" implies that it should be rejected.

Reply:
Lets open a ticket to handle this in the future. 
Currently what will happen is if an invalid data is decided, a node might reject it and fail.

Only way this can happen, I think, is if there was a fork and the node didn't update (because a majority did sign it after all)

Reply:
There are 2 scenarios a node will get an invalid decided message
1) malicious cluster majority
2) fork, didn't update.

In both cases the node should reject I think

Reply:
Added #201